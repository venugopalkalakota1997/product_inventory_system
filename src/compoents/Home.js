import React from 'react';



class Home extends React.Component {
    state = {}
    render() {
        return (
            <div>    
                <h3 > Home Page Component </h3>               
            </div>
        );
    }
}

export default Home;