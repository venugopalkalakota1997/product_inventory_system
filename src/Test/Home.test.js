import React from 'react';
import { render } from '@testing-library/react';
import Home from '../compoents/Home'
it('renders without crash',()=>{
  render(<div></div>)
})