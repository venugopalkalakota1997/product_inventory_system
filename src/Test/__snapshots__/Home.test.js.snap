// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mathes Snapshot 1`] = `
<div
  className="home"
>
  <input
    className="searchtext"
    name="search"
    onChange={[Function]}
    placeholder="Search.."
    type="text"
  />
  <button
    className="searchbutton"
    onClick={[Function]}
    type="submit"
  >
    Search
  </button>
  <br />
  <br />
  <span>
    <div
      className="filter"
    >
      <p>
        Filter by:
      </p>
      <select
        id="productcategory"
        onChange={[Function]}
      >
        <option
          id="productcategory"
        >
          Select Category
        </option>
        <option
          id="productcategory"
        >
          Televison
        </option>
        <option
          id="productcategory"
        >
          Mobile 
        </option>
        <option
          id="productcategory"
        >
          Furniture
        </option>
        <option
          id="productcategory"
        >
          Computer Accessories
        </option>
      </select>
       â€ƒ
      <label>
        Product in stock :
        <input
          defaultChecked={false}
          onChange={[Function]}
          type="checkbox"
        />
        Yes
      </label>
    </div>
  </span>
  <br />
  <br />
</div>
`;
